database/index.ts

```diff
@@ -1,14 +1,27 @@
 import { DatabasePool, sql } from 'slonik';
 import { 
-	createPool, 
+	createPool,
+    type Interceptor,
+    type QueryResultRow,
+    SchemaValidationError,
+    createTypeParserPreset
 } from 'slonik';
+import {
+  createFieldNameTransformationInterceptor
+} from 'slonik-interceptor-field-name-transformation';
 
 const DB_URL = process.env.DB_URL || 'postgres://postgres:postgres@db:5432/postgres';
 const CA_CERT = process.env.CA_CERT
 let pool: DatabasePool;
 
 const baseConfig = {
-  typeParsers:[]
+  interceptors: [
+    createFieldNameTransformationInterceptor({
+      format: 'CAMEL_CASE'
+    }),
+    createResultParserInterceptor()
+  ],
+  typeParsers:[...createTypeParserPreset()]
```

Comment:
I think this line is the source of your ISO8601 date format problem returning. Check the slonik docs but I believe the TypeParserPreset has a parser for timestamptz->integer not ISO8601